src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/eye_ref.cl

```diff
@@ -0,0 +1,26 @@
+
+#include "include/batch_headers/fetch_data.cl"
+
+KERNEL(eye_ref)(const __global INPUT0_TYPE* input, __global OUTPUT_TYPE* C) {
```

Comment:
Why output name is "C"? Does it have some meaning? If not, maybe rename it to something more obvious?

Reply:
done